### ISSUE TITLE
Play from a websocket stream

### ISSUE THREAD

**Issue by quartexNOR** [BMSVieira/moovie.js#10]
I have implemented a service for a "filesystem over websocket", which is a service that accepts calls in the form of JSON messages. So you have functions like "exists", "read", "write" etc etc all implemented as JSON RPC over websocket. The "real" filesystem just being the users home folder -- so that i can access that from https/ws remotely.

Would it be possible to use your video player to handle data from websocket?

For example, if i could have an event like this in my websocket client:

on["data"] = function (tagId, streamId, bytes) {
  mooviePlayer.buffer.push (bytes);
}

Basically, pushing data into a buffer that the movieplayer is streaming from, then that would make moovie.js by par the most versatile and cool movie player out there.

Hope you think about this, im not the only one having to use WSS for data transport

**Comment by BMSVieira** [comment]
Thank you, I'll take a look.
